concourse/tasks/smoke-test.yml

```diff
@@ -0,0 +1,19 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/curl-ssl
+params:
+  URL:
+  MESSAGE:
+run:
+  path: sh
+  args:
+    - '-c'
+    - |
+      set -eu
+
+      echo "$MESSAGE"
+
+      # TODO we can come up with a more thorough test than this
```

Comment:
assuming this is captured on a backlog somewhere?

Reply:
yes there is a ticket for this - to be fair this again taken from another service in the covid19 space